srcpkgs/gajim/template

```diff
@@ -1,19 +1,25 @@
 # Template file for 'gajim'
 pkgname=gajim
-version=1.2.2
-revision=3
+version=1.3.1
+revision=1
+wrksrc=gajim-gajim-${version}
 build_style=python3-module
 hostmakedepends="gettext python3-setuptools"
 depends="python3-gobject python3-nbxmpp python3-pyasn1 python3-setuptools
  python3-precis-i18n python3-keyring python3-cssutils python3-packaging
- python3-css-parser farstream python3-dbus libsoup"
+ python3-css-parser python3-idna farstream python3-dbus libsoup"
 short_desc="Full featured Jabber/XMPP client"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-3.0-only"
 homepage="https://www.gajim.org"
-changelog="https://dev.gajim.org/gajim/gajim/-/raw/master/ChangeLog"
+changelog="https://dev.gajim.org/gajim/gajim/raw/gajim-${version}/ChangeLog"
```

Comment:
```suggestion
changelog="https://dev.gajim.org/gajim/gajim/raw/gajim_${version%.*}/ChangeLog"
```

To get the branch for those releases.

Reply:
I wonder, why you suggest to use a variable, which is not working, if version is is like `1.4`. My proposal should work with every update and point to the most recent Changelog. What do I miss here?

Reply:
Because I like using changelogs to see what the next version provides ;)

But you make a reasonable point, so merging. Thanks.